UnityProject/Assets/Scripts/Medical/HealthScanner.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealthScanner : PickUpTrigger
```

Comment:
I think it would be a good idea to keep the naming consistent between prefab and script, so either call this MedicalScanner or change the name of the prefab to HealthScanner